city_scrapers/spiders/det_transportation.py

```diff
@@ -0,0 +1,16 @@
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.spiders import CityScrapersSpider
+
+from city_scrapers.mixins import DetCityMixin
+
+
+class DetTransportationSpider(DetCityMixin, CityScrapersSpider):
+    name = "det_transportation"
+    agency = "Detroit Department of Transportation"
+    dept_cal_id = '166'
+    agency_cal_id = 'All'
+    dept_doc_id = '166'
+    agency_doc_id = 'None'
+
```

Comment:
To avoid the issue with all day events you can override the `parse_event_page` method to check and see if a page has a valid start time and ignore it if it doesn't. It would go something like this

```python
def parse_event_page(self, response):
  try:
    self._parse_start(response)
  except AttributeError:
    return
  return super().parse_event_page(response)
```